fetch/wire_types.go

```diff
@@ -35,21 +35,19 @@ type ResponseBatch struct {
 }
 
 type MeshHashRequest struct {
```

Comment:
It took me a long time to understand what By really meant, I think a comment here would be useful to any future reader.

```suggestion
// The MeshHashRequest is used by the ForkFinder to request sets of hashes from
// a peer that are used to determine the point at which a divergence occured in
// this node and the peer's history. From and To define the beginning and end
// hash of the range of hashes to request and in the case that the length of
// that range exceeds Config.MaxHashesInReq, instead of requesting every hash
// we sample Config.MaxHashesInReq hashes across the range and By is set to the
// Step used in sampling. Eg if By is 2 we will request every other hash.
type MeshHashRequest struct {
```

Reply:
Updated description